src/cargo/ops/lockfile.rs

```diff
@@ -43,13 +43,23 @@ pub fn write_pkg_lockfile(ws: &Workspace<'_>, resolve: &Resolve) -> CargoResult<
 
     let mut out = String::new();
 
-    // Preserve the top comments in the lockfile
-    // This is in preparation for marking it as generated
-    // https://github.com/rust-lang/cargo/issues/6180
+    // At the start of the file we notify the reader that the file is generated.
+    // Specifically Phabricator ignores files containing "@generated", so we use that.
+    let marker_line = "# This file is automatically @generated.  Not intended for manual editing.";
```

Comment:
Excuse me, why there is two spaces here?

Reply:
https://en.wikipedia.org/wiki/Sentence_spacing